server/questions.js

```diff
@@ -130,18 +135,31 @@ var validateQuestionByType = function(question, type){
 			result = multipleChoiceValidator(question);
 			break;
 
+		case common.questionTypes.TRUEFALSE.value:
+			result = trueAndFalseValidator(question);
+			break;
+
 		default:
 			break;
 	}
 	return result;
 }
 
 var multipleChoiceValidator = function(question){
-
-	if (question.choices.length < 2){
-		return 'Need two or more options for Multiple Choice Question';
+	if (question.choices){
```

Comment:
Combine the 2 ifs into one. If not choices or choices.lengeth more than two. Same for the tf validate